liquid-derive/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "liquid-derive"
+version = "0.18.0"
+authors = ["Johann Hofmann <mail@johann-hofmann.com>"]
```

Comment:
This is all you, feel free to put your name in here

Never have found good guidance on `authors`.  At what point do we add ourselves?  Remove Johann?

Reply:
In my opinion you should have added yourself since the moment you took on as an owner of this project (I mean, you do most contributions and you're the one who deploys the new versions of the crate). You can have more than one author listed in the TOML, so there is no need to delete Johann.